packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -256,7 +257,10 @@ class VMService {
 
   /// Whether our connection to the VM service has been closed;
   bool get isClosed => _peer.isClosed;
-  Future<Null> get done => _peer.done;
+
+  Future<Null> get done async {
```

Comment:
`Future<void>`?

Reply:
I really don't know if we're returning `null` or `void` here.